templates/keras_cv/index.md

```diff
@@ -35,32 +35,28 @@ import tensorflow as tf
 from tensorflow import keras
 import tensorflow_datasets as tfds
 
-preprocessing_layers = [
-    keras_cv.layers.RandomResizedCrop(
```

Comment:
Why is this layer dropped?

Reply:
we're changing the API soon and I don't want to push the layer anymore until the API is done.